integrations/GoogleTagManager/browser.js

```diff
@@ -53,10 +53,6 @@ class GoogleTagManager {
       eventName = "Viewed " + pageCategory + " " + pageName + " page";
     }
 
-    if(!eventName) {
```

Comment:
is this not required ?
present in master https://github.com/rudderlabs/rudder-sdk-js/blob/master/integrations/GoogleTagManager/browser.js#L56